app/1.0/start/toolbox/add-elements.md

```diff
@@ -1,84 +1,71 @@
 ---
-title: Step 3. Add some elements
+title: Step 3. Add an element
 subtitle: "Build an app with App Toolbox"
 ---
 
 <!-- toc -->
 
-Now that you've added a new view to your application, you can start building
-out the details of that view.
```

Comment:
This deleted sentence actually introduces the step—so some variant of this should probably stay here. (And we should keep the term "view" which is used throughout the tutorial).